src/libs/actions/RequestConflictUtils.ts

```diff
@@ -0,0 +1,23 @@
+import type {WriteCommand} from '@libs/API/types';
+import type OnyxRequest from '@src/types/onyx/Request';
+import type {ConflictActionData} from '@src/types/onyx/Request';
+
+function resolveDuplicationConflictAction(persistedRequests: OnyxRequest[], commandToFind: WriteCommand): ConflictActionData {
```

Comment:
Can you please add a plain English explanation for this method and why it exists so it's easier to follow for other devs? Thanks!